src/rules/__tests__/prefer-importing-jest-globals.test.ts

```diff
@@ -0,0 +1,168 @@
+import { TSESLint } from '@typescript-eslint/utils';
+import dedent from 'dedent';
+import rule from '../prefer-importing-jest-globals';
+import { espreeParser } from './test-utils';
+
+const ruleTester = new TSESLint.RuleTester({
+  parser: espreeParser,
+  parserOptions: {
+    ecmaVersion: 2015,
+  },
+});
+
+ruleTester.run('prefer-importing-jest-globals', rule, {
+  valid: [
```

Comment:
maybe add a case where method _is_ imported, but not from `@jest/globals`? e.g. `const test = require('my-test');`. But I guess that's probably handled by the `parseJestCall` utility?

Reply:
Added